_includes/v22.1/performance/lease-preference-system-database.md

```diff
@@ -0,0 +1,8 @@
+To reduce latency while making online schema changes, we recommend specifying a `lease_preference` on the `system` database to a single region and running all subsequent schema changes from a node within that region. For example, if the majority of online schema changes come from machines that are geographically close to `us-east1`, run the following:
```

Comment:
This looks really good! Two small things:

First, suggest making first mention of 'online schema changes' a link to https://www.cockroachlabs.com/docs/stable/online-schema-changes.html (appropriate to the docs version of course)

Since this is also going on the schema changes page, you can use the Liquid syntax like the below to make it not a link on that page  (ex is taken from https://github.com/cockroachdb/docs/commit/f4cc14a2013bdd8baf3a08feed883fe516a7f7c7)

```
{% if page.name == "multiregion-overview.md" %} using the higher-level multiregion abstractions, you must use Super Regions {% else %} using the higher-level [multiregion abstractions](multiregion-overview.html), you must use [Super Regions](multiregion-overview.html#super-regions) {% endif %}
```

Second, suggest adding a link to https://www.cockroachlabs.com/docs/stable/configure-replication-zones.html somewhere in your mention of `lease_preference`, e.g. "we recommend specifying the `lease_preference` [zone configuration](configure-replication-zones.html) ..."